lib/src/source/hosted.dart

```diff
@@ -329,7 +329,7 @@ class BoundHostedSource extends CachedSource {
           stackTrace);
     }
 
-    if (error is io.SocketException) {
+    if (error is io.SocketException || error is io.TlsException) {
       fail("Got socket error trying to find package $package at $url.", error,
```

Comment:
It would be good to have a more specific error message, like "Got TLS error ..." if it's a TlsException.

Reply:
Done.